docs/components/vectordbs/dbs/milvus.mdx

```diff
@@ -0,0 +1,35 @@
+[Milvus](https://milvus.io/) Milvus is an open-source vector database that suits AI applications of every size from running a demo chatbot in Jupyter notebook to building web-scale search that serves billions of users.
+
+### Usage
+
+```python
+import os
+from mem0 import Memory
+
+config = {
+    "vector_store": {
+        "provider": "milvus",
+        "config": {
+            "collection_name": "test",
+            "embedding_model_dims": "123",
+            "url": "127.0.0.1",
+            "token": "8e4b8ca8cf2c67",
+        }
+    }
+}
+
+m = Memory.from_config(config)
+m.add("Likes to play cricket on weekends", user_id="alice", metadata={"category": "hobbies"})
+```
+
+### Config
+
+Here's the parameters available for configuring Mlvus Database:
+
+| Parameter | Description | Default Value |
+| --- | --- | --- |
+| `url` | Full URL for Milvus/Zilliz server | `http://localhost:19530` |
+| `token` | Token for Milvus/Zilliz server | `None` |
```

Comment:
You should properly clarify here that token is required only for Zilliz sever. Please correct me if I'm wrong here

Reply:
Here is a description of how to set the uri and token. you can directly copy and paste it here. https://milvus.io/docs/build-rag-with-milvus.md#Load-data-into-Milvus
![image](https://github.com/user-attachments/assets/a01f92ab-77dc-4ff5-98fb-0e1b079e374b)

BTW, this parameter in milvus/zilliz is named as `uri`, instead of `url`, but  this [config page](https://docs.mem0.ai/components/vectordbs/config#master-list-of-all-params-in-config)  shows the `url` is a suggested param name in mem0 across all vector dbs. So it is a little bit conflicted, @Dev-Khant do you have any suggestion, which one is better?

Reply:
For now, we can use `url`, if users face many issues with the config then we can modify it.